apps/codeforafrica/src/lib/api.netlify-cms/mappings.js

```diff
@@ -0,0 +1,22 @@
+const Project = {
```

Comment:
Not sure about this mapping file structure e.g. why do we need `title`, etc. My current thinking would be to remain close to the Netlify CMS nomenclature i.e. collections. Something like
```js
export const markdownContent = {
  projects: {
    "*": {
      subtitle: true,
    },
  },
  pages: {
    index: {
      "hero.title": true,
    },
  },
}
```

1. `projects` and `pages` are **collections**
2. `*` and `index` are slug for _folder_ and _file_ collection respectively (`*` means applicable to all slugs since all items in a folder collection have the same structure, while `index` means match `index` slug only), and
3. "hero.title" means title in hero object obviously.

We can use JS's optional chaining to convert that string e.g.
```js
var indexPage = { hero: { title: "Civic Tech Heroes" } }; 
function getObjectField(object, path) {
  return path.split(".").reduce((acc, curr) => acc?.[curr], object);
}

console.log(getObjectField(indexPage, "hero.title")); // Civic Tech Heroes
```